package.json

```diff
@@ -42,10 +42,8 @@
     "babel-preset-env": "^1.1.8",
     "babel-preset-stage-2": "^6.22.0",
     "babel-register": "^6.23.0",
-    "cli-tester": "^2.0.0",
     "codecov": "^1.0.1",
     "commitizen": "^2.9.6",
-    "concurrently": "^3.3.0",
```

Comment:
`nps-utils` brings this in for us and manages finding the right binary for us :)